packages/sdk-components-react/src/span-container.ws.tsx

```diff
@@ -0,0 +1,26 @@
+import { PaintBrushIcon } from "@webstudio-is/icons/svg";
+import {
+  defaultStates,
+  type PresetStyle,
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import { span } from "@webstudio-is/react-sdk/css-normalize";
+import type { defaultTag } from "./span-container";
+import { props } from "./__generated__/span.props";
+
+const presetStyle = {
+  span,
+} satisfies PresetStyle<typeof defaultTag>;
+
+export const meta: WsComponentMeta = {
+  type: "container",
+  label: "Span",
```

Comment:
Shouldn't the label be "Text"?

Reply:
I mean when we automatically wrap text with a span, in the navigator it probably should say "Text", same as we wanted to rename " Styled Text" to "Text" in rte when clicking on style button

Reply:
Then we will have 3 text components, no?
Text which is div, text with is span, text which is rich-text-child span.

Reply:
Hmm, right now Text has tag property, so it's possible to have one component and render as different tags but I am not sure what's best here since we still deciding on the strategy for multi-tag components